code/modules/research/designs/tool_designs.dm

```diff
@@ -104,7 +89,6 @@
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING
 
-
```

Comment:
![image](https://user-images.githubusercontent.com/82386923/212966660-96f8a9a3-05bd-4138-8737-acb4220747d0.png)

Reply:
?

Reply:
I think the question was, why removing the empty line here.

Reply:
Keeping resolved - i removed the SR edit and upstream doesnt have an extra here